<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="comSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openCOM">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCom">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbStopBit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbBaudRate">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbCheckBit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>no</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>space</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mark</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbDataBit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="receiveSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>接收区设置</string>
    </property>
    <widget class="QCheckBox" name="ccbToFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>接收转向文件</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ccbHex">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>十六进制显示</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ccbAutoReturn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>自动换行</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ccbPause">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>暂停接收</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSaveData">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>保存数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbClearData">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>清除数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sendSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>发送区设置</string>
    </property>
    <widget class="QCheckBox" name="ccbLoopSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>数据循环发送</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>发送间隔:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>毫秒</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leSendInterval">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>2000</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="receiveBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>串口数据接收</string>
    </property>
    <widget class="QTextEdit" name="teDataReceived">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>361</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="log">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>日志</string>
    </property>
    <widget class="QTextEdit" name="teLog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sendBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>串口数据发送</string>
    </property>
    <widget class="QTextEdit" name="teDataSend">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbSendButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>200</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
